pages/style-guide/html.md

```diff
@@ -4,64 +4,110 @@
 
 This page outlines the style guide for HTML pages in all jQuery projects. These rules apply to web sites, demo pages, inline examples, test pages, etc. Exceptions are allowed for pages that must violate the rules by their very nature, e.g., a page that tests XHTML.
 
-## Doctype
-Always use the minimal, versionless doctype.
+## Linting
 
-```
-<!doctype html>
-```
+The markup should be valid HTML5 and it must pass the automated testing we use via grunt-html.
 
-## Language
+## Spacing
 
-Always define which language the page is written in.
+In general, the jQuery style guide encourages liberal spacing for improved human readability.
 
-```
+- Identation only with tabs where one tab indents 4 spaces. Don't indent the direct children of `html`, `body`, `script`, or `style`. Indent everything else.
+- Don't leave whitespace at the end of lines.
+- Separate attributes with a space.
+
+```html
+<!doctype html>
 <html lang="en">
-```
+<head>
+	<meta charset="utf-8">
+	<title>Sample Page</title>
 
-## Encoding
+	<link rel="stylesheet" href="/style.css">
+	<style>
+	body {
+		font-size: 100em;
+	}
+	</style>
+
+	<script src="/jquery.js"></script>
+	<script>
+	$( function() {
+		$( "p" ).text( $.fn.jquery );
+	});
+	</script>
+</head>
+<body>
 
-Always define the character encoding. The encoding should be defined as early as possible.
+<section>
+	<p>jQuery is awesome!<p>
+</section>
 
-```
-<meta charset="utf-8">
+</body>
+</html>
 ```
 
-## Elements and Attributes
-
-All element and attribute names should be lowercase. Attribute values should be quoted. Optional closing tags should be included. Self-closing elements should not be closed. Optional attributes should be omitted. Always include `html`, `head`, and `body` tags.
+## Formatting
+
+ - Always use the minimal, versionless and lowercase doctype.
+ - Always define which language the page is written in.
+ - Always include `html`, `head`, and `body` tags.
+ - Always define the character encoding. The encoding should be defined as early as possible and must be in the [first 1024 bytes of the document](https://html.spec.whatwg.org/multipage/semantics.html#charset).
+ - Always use lowercase names for elements and attributes.
+ - Always quote attribute values. Always use double quotes instead of single quotes.
+ - Include the optional closing tags.
+ - Self-closing elements should not be closed.
+ - Optional attributes should be omitted.
+ - `rel`, `type`, `src`, `href` and `class` should be written before every other attribute.
+ - Always include `alt` attribute for images.
+
+```html
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Good Example</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="/jquery.js"></script>
+</head>
+<body>
 
-*   No `type` or `language` attributes on `script` tags.
-*   No `type` attribute on `link` or `style` tags.
+<img class="sample-image" src="assets/img/img.png" alt="Sample Image" width="200px" height="200px">
```

Comment:
Why are you including `width` and `height` here? This violates separation of concern.